src/useQueryString.js

```diff
@@ -2,30 +2,31 @@ import React from 'react'
 import qs from 'query-string'
 
 export const defaultOptions = {
+  /** @type {import('query-string').ParseOptions} */
   parse: {
     arrayFormat: 'bracket'
   },
+  /** @type {import('query-string').StringifyOptions} */
   stringify: {
     skipEmptyString: true, 
     skipNull: true, 
     arrayFormat: 'bracket'
   }
 }
 
-const QueryStringContext = React.createContext({ update: replaceQueryString, search: null, options: defaultOptions })
+/** @type {React.Context<object>} */
+export const QueryStringContext = React.createContext({ update: replaceQueryString, search: null, options: defaultOptions })
```

Comment:
Wat mij betreft mag deze gewoon met een kleine letter:

```suggestion
export const queryStringContext = React.createContext({ update: replaceQueryString, search: null, options: defaultOptions })
```